### PR TITLE
reintroduce obj_na_str after regression. vbump

### PR DESCRIPTION
**Pull request by gmbecker**
[skip vbump]

### CODE REVIEW

File: R/generics.R [hunk 1]
```diff
@@ -285,6 +285,21 @@ setMethod("obj_format<-", "ANY", function(obj, value) {
   obj
 })
 
+#' @rdname lab_name
```
Comment (shajoezhu) [inline R/generics.R:288]: Hi @gmbecker , it took me a while to guess what "lab_name" means, I guess it is 'label_name'? I was wondering if this what you meant? if it is, we can spin name another issue and replace it to ‘label_name'?

**Comment by shajoezhu** [comment]
Hi @gmbecker , I was wondeirng if there is some background issue I could refering this to? Thanks

File: R/generics.R (hunk 1 above)
Comment (gmbecker) [reply R/generics.R:288]: this only controls the physical name of the Rd file within the directory structure, so it really doesn't matter what it is (and thus doesn't seem worth changing)

**Comment by gmbecker** [comment]
@cicdguy the linter blocked merging because it makes changes but skips actions which means its impossible to meet the requirements

File: R/generics.R [hunk 2]
```diff
@@ -498,7 +499,8 @@ setGeneric("table_inset<-", function(obj, value) standardGeneric("table_inset<-"
 
 #' @rdname table_inset
 #' @export
-setMethod("table_inset<-", "MatrixPrintForm",
+setMethod(
+  "table_inset<-", "MatrixPrintForm",
```
Comment (gmbecker) [inline R/generics.R:503]: @cicdguy in paticular, this change that the linter is trying to force is actively bad and I do not want it.

File: R/generics.R (hunk 1 above)
Comment (shajoezhu) [reply R/generics.R:288]: hi Gabe, this is just a minor comment, definitely not blocking this PR. I was just thinking if there are ways that we could improve, and make the code easier to read. :)

**Review by shajoezhu — APPROVED** [review]
lgtm! thanks @gmbecker

**Pull request merged**